.github/workflows/ci.yml

```diff
@@ -71,6 +71,30 @@ jobs:
         with:
           name: test-logs
           path: build/host/Testing/Temporary/
+  host_build_test_clang_msvc:
+    name: Host Build & Test (clang-msvc)
+    runs-on: ubuntu-latest
+    container: ghcr.io/philips-software/amp-devcontainer-cpp:5.1.4@sha256:46239906460dedb3baf3c33d9275f3de4f17d7a237fc136c2013b021589a6dbd # 5.1.4
+    steps:
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+        with:
+          persist-credentials: false
+      - uses: actions/cache@0c45773b623bea8c8e75f6c82b208c3cf94ea4f9 # v4.0.2
+        id: cache-winsdk
+        with:
+          path: /winsdk
+          key: cache-winsdk
+      - if: ${{ steps.cache-winsdk.outputs.cache-hit != 'true' }}
+        run: xwin --accept-license splat --preserve-ms-arch-notation --include-debug-libs && mv .xwin-cache/splat/ /winsdk
+      - uses: hendrikmuhs/ccache-action@ed74d11c0b343532753ecead8a951bb09bb34bc9 # v1.2.14
+        with:
+          key: ${{ github.job }}-clang-msvc
```

Comment:
Please verify that this cache works. For our other windows builds via amp-devcontainer we had to add the following key to the `Windows` configure preset:

```
      "environment": {
        "CCACHE_DEPEND": "true",
        "CCACHE_COMPILERTYPE": "clang-cl"
      },
```

Without this key ccache couldn't cache. It might have been fixed in later ccache//clang-cl versions though.

Reply:
After trying out, it seems to work